src/Header/Contact/style.css

```diff
@@ -1,4 +1,49 @@
-.contact-page-content{
-    padding: 10px;
+.contact-page-content {
     margin-top: 20px;
+    margin-left: 45px;
+}
+
+.form-email-text-submit {
+    display: flex;
+    flex-flow: column wrap;
+    gap: 25px;
+    margin-left: 450px;
+    margin-top: 70px;
+}
+
+#email-id-box {
+    padding: 8px;
+    background-color: black;
+    border: 1px solid grey;
+    border-radius: 10px;
+    width: 250px;
+    color: #fff
+}
+
+#subject-text-input-box {
+    padding: 8px;
+    background-color: black;
+    border: 1px solid grey;
+    border-radius: 10px;
+    width: 250px;
+    color: #fff
+}
+
+#enter-text-here-input-box {
+    padding: 8px;
+    background-color: black;
+    border: 1px solid grey;
+    border-radius: 10px;
+    width: 250px;
+    height: 200px;
```

Comment:
You can decrease the height of the box a bit I think around 150 would be fine it looks bit long?